test/commands-test.js

```diff
@@ -113,111 +124,132 @@ describe('window commands', function () {
   })
 
   describe('waitUntilWindowLoaded()', function () {
-    it('waits until the current window is loaded', function () {
-      return app.client.waitUntilWindowLoaded()
-        .webContents.isLoading().should.eventually.be.false
+    it('waits until the current window is loaded', async function () {
+      await app.client.waitUntilWindowLoaded()
+      return app.webContents.isLoading().should.eventually.be.false
     })
   })
 
   describe('browserWindow.isMaximized()', function () {
-    it('returns true when the window is maximized, false otherwise', function () {
-      return app.browserWindow.isMaximized().should.eventually.be.false
-        .browserWindow.maximize().waitUntil(function () {
-          // FIXME window maximized state is never true on CI
-          if (process.env.CI) return Promise.resolve(true)
-
-          return this.browserWindow.isMaximized()
-        }, 5000).then(function () { })
+    it('returns true when the window is maximized, false otherwise', async function () {
+      const notMaximized = await app.browserWindow.isMaximized()
+      expect(notMaximized).to.equal(false)
+      await app.browserWindow.maximize()
+      var maximized = await app.browserWindow.isMaximized()
```

Comment:
```suggestion
      const maximized = await app.browserWindow.isMaximized()
```

Reply:
We reassign this below, so cannot use `const` but will use `let`!